content/docs/work-with-source-git.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Source-git"
+date: 2020-06-25T23:59:59+01:00
+draft: false
+chapter: false
+weight: 2
+aliases:
+- /source-git
+- /omfg
```

Comment:
:see_no_evil: :hear_no_evil: :speak_no_evil:

Reply:
I'm not sure these aliases are working... I cannot seem to get to this page when running the page locally with `make server`.

Reply:
yes, I spent like 30 minutes on that and finally figured it out, this line is just a leftover :D thanks for noticing